gradle.properties

```diff
@@ -17,7 +17,8 @@ grizzlyVersion=2.25.1
 gsonVersion=2.8.9
 includePravegaCredentials=true
 jacksonVersion=2.9.10.3
-junitVersion=4.12
+#junitVersion=4.12
```

Comment:
Please remove this line
#junitVersion=4.12

Reply:
Done